src/openrct2-ui/windows/Park.cpp

```diff
@@ -582,8 +583,11 @@ static void window_park_set_disabled_tabs(rct_window* w)
 
 static void window_park_prepare_window_title_text()
 {
-    set_format_arg(0, rct_string_id, gParkName);
-    set_format_arg(2, uint32_t, gParkNameArgs);
+    auto& park = OpenRCT2::GetContext()->GetGameState()->GetPark();
+    auto parkName = park.Name.c_str();
+
+    set_format_arg(0, rct_string_id, STR_STRING);
+    set_format_arg(2, const char*, parkName);
```

Comment:
This may be what causes some of the backtrace reports. `parkName` points to a temporary buffer returned by `c_str` that is not guaranteed to exist after this function call. Forcing this function to be inlined and calling it just before `gCommonFormatArgs` gets used may fix it.

Reply:
Thanks for spotting this, I will try to resolve...

Reply:
On closer inspection of the disassembly, this is not a temporary. It is actually using the internal buffer of the std::string on the global Park instance. The std::string is not copied.

Reply:
Okay, I found the cause. It is because the price of the park is overwriting the park name args in the arg buffer.

`set_format_arg(6, uint32_t, parkEntranceFee);`